packages/modeldb/src/utils/AbstractSqliteDB.ts

```diff
@@ -0,0 +1,24 @@
+import { SqlitePrimitiveValue } from "./encoding.js"
+
+export abstract class AbstractSqliteDB {
+	// exec ? or maybe always prepare
```

Comment:
comment

Reply:
i think having a method/query class is fine, this was a note to self and should not have been commited!